### PR TITLE
Complete British English translation and correct typos in the core US English locale.

### PR DESCRIPTION
**Pull request by Enverex**
Completes the en-GB translation and fixes some minor issues in the core Neos default locale.

### CODE REVIEW

**Comment by ProbablePrime** [comment]
Wow thank you for the fixes :D

File: en.json [hunk 1]
```diff
@@ -250,15 +250,15 @@
         "Options.SeatedMode.Off" : "Seated Mode: Off",
 
         "Indicator.OnlineUsers" : "Online Users: ~{num_total} ({num_registered})",
-        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, Good {Good} Slow {Slow} Down {Down} NoInternet {No Internet}} ({response_ms} ms)",
+        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, Good {Good} Slow {Slow} Down {Down} No Internet {No Internet}} ({response_ms} ms)",
```
Comment (Frooxius) [inline en.json:253]: This needs to be "NoInternet" it's a keyword match, with the actual displayed message being in the braces { }

File: en.json [hunk 2]
```diff
@@ -377,7 +377,7 @@
         "Undo.BakeMeshes" : "Bake {name}",
         "Undo.ChangeMesh" : "Change Mesh",
         "Undo.InsertParent" : "Insert Parent for {name}",
-        "Undo.Wiggler" : "Evil Stuff",
+        "Undo.Wiggler" : "Wiggler",
```
Comment (Frooxius) [inline en.json:380]: Please don't change this, it's an easter egg! xP

**Comment by Frooxius** [comment]
Thank you for the contributions!

File: en.json (hunk 2 above)
Comment (Enverex) [reply en.json:380]: I had my suspicions ;)

File: en.json (hunk 1 above)
Comment (Enverex) [reply en.json:253]: My mistake, I thought it was the other way around (with the braces being the actual "variable/keyword").

**Pull request merged**